manifests/m/Microsoft/PowerAutomateDesktop/2.26.139.22312/Microsoft.PowerAutomateDesktop.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Automatically updated by the winget bot at 2022/Nov/15
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.PowerAutomateDesktop
+PackageVersion: 2.26.139.22312
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://flow.microsoft.com
+PublisherSupportUrl: https://flow.microsoft.com/en-us/support/
+PrivacyUrl: https://privacy.microsoft.com/en-us/PrivacyStatement
+Author: Microsoft Corporation
+PackageName: Power Automate Desktop

```

```suggestion
PackageName: Power Automate for desktop
```
Must match Control Panel